apps/c/redis/features.txt

```diff
@@ -7,3 +7,4 @@ fs
 net
 pipe
 epoll
+poll
```

Comment:
Does `Redis` need poll() ?

Reply:
I'm not familiar with Redis, but it will be a error if deleted the `poll` feature.The part of error message is: